klarna-on-demand/src/main/java/com/klarna/ondemand/crypto/SharedPreferencesCryptoImpl.java

```diff
@@ -62,21 +38,10 @@ private CryptoSharedPreferencesBaseImpl(android.content.Context context) throws
         publicKeyBase64Str = toBase64(publicKey);
     }
 
-    @Override
-    public String getPublicKeyBase64Str() {
-        return publicKeyBase64Str;
-    }
-
-    protected PrivateKey getPrivateKey() {
-        return privateKey;
-    }
-
-    @Override
-    public String sign(String message) throws NoSuchAlgorithmException, InvalidKeyException, SignatureException {
-        Signature sign = Signature.getInstance(DIGEST_ALGORITHM);
-        sign.initSign(getPrivateKey());
-        sign.update(message.getBytes());
-        return new String(Base64.encode(sign.sign(), Base64.DEFAULT));
+    private static SharedPreferences getSharedPreferences(Context context) {
+        return context.getSharedPreferences(
+                    context.getPackageName() + KLARNA,
```

Comment:
`context.getPackageName() + KLARNA` -> `context.getPackageName()` 
(well, the name Klarna appears in the package name).